tests/src/tests/utils.rs

```diff
@@ -33,3 +33,13 @@ fn pip_borsch_deserialization_selected_values(#[case] serialized: &[u8], #[case]
     let deserialized: Pips = borsh::from_slice(serialized).unwrap();
     assert_eq!(deserialized, Pips::from_pips(pips).unwrap());
 }
+
+pub trait AsNearSdkLog {
+    fn as_near_sdk_log(&self) -> String;
+}
```

Comment:
Maybe make it a part of `defuse-near-utils` crate and re-use in the `SimulateInspector::into_report`?
```suggestion
pub trait NearSdkLog {
    fn to_near_sdk_log(&self) -> String;
}
```

PS: ideally, add this `pub fn to_log(&self) -> String()` method on `near-sdk` level via PR
PPS: ideally, this should be a trait, so we can use it in generic code, but that would require too many changes on `near-sdk` level that can potentially break backwards-compatibility

Reply:
> PS: ideally, add this pub fn to_log(&self) -> String() method on near-sdk level via PR
PPS: ideally, this should be a trait, so we can use it in generic code, but that would require too many changes on near-sdk level that can potentially break backwards-compatibility


created separate task for it ✔️